java/flight/src/test/java/org/apache/arrow/flight/TestBackPressure.java

```diff
@@ -41,6 +41,7 @@
   /**
    * Make sure that failing to consume one stream doesn't block other streams.
    */
+  @Ignore
```

Comment:
can you document why  it is ignored and a JIRA to re-enable it?

Reply:
Since there is ARROW-4765 whoever works on that can deal with disabled unit test